File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution(object):
    def isPalindrome(self, s):
        position = 0
        reVal = 0
        sStripped = []
        for i in s:
            if i.isalpha() == True or i.isnumeric() == True:
                sStripped.append(i.lower())

        for position in range(len(sStripped)/2):
            if sStripped[position] == sStripped[(len(sStripped)-1)-position]:
                reVal += 1
        
        if reVal == len(sStripped)/2:
            return True
        
        else:
            return False
        

File: 0125-valid-palindrome/0125-valid-palindrome.go
func strip(s []byte) []byte {
    n := 0
    for _, b := range s {
        if ('a' <= b && b <= 'z') ||
            ('0' <= b && b <= '9'){
            s[n] = b
            n++
        }
    }
    return s[:n]
}

func isPalindrome(s string) bool {
    var str = strip([]byte(strings.ToLower(s)))
	lp, rp := 0, len(str) - 1
	for lp < rp {
		if str[lp] != str[rp] {
            return false
		}
        lp++
        rp--
	}
	return true
    
}